src/runtime/server/settings.ts

```diff
@@ -33,15 +88,31 @@ export type SettingsInput = {
    * production, without some kind of security/access control, you will almost
    * certainly want this disabled.
    *
-   * To learn more about GraphQL Playgorund see
+   * To learn more about GraphQL Playground see
    * https://github.com/prisma-labs/graphql-playground
    */
   playground?: boolean | PlaygroundSettings
+  /**
+   * Enable CORS for your server
+   *
+   * When true is passed, the default config is the following:
+   *
+   * ```
+   * {
+   *   "origin": "*",
+   *   "methods": "GET,HEAD,PUT,PATCH,POST,DELETE",
+   *   "preflightContinue": false,
+   *   "optionsSuccessStatus": 204
+   * }
```

Comment:
This could be a pretty dangerous default.

💡

I think a future `"strict"` mode would be nice for teams that want more safety. With strict mode Nexus would warn in dev mode that user should explicitly set options or disable CORS. In production it would crash.